client/profile-wizard/index.js

```diff
@@ -243,28 +239,12 @@ class ProfileWizard extends Component {
 			} ),
 		];
 
-		let redirectUrl = null;
-		if ( shouldConnectJetpack ) {
-			promises.push(
-				getJetpackConnectUrl( {
-					redirect_url: getAdminLink( 'admin.php?page=wc-admin' ),
-				} ).then( ( jetpackConnectUrl ) => {
-					const error = getPluginsError( 'getJetpackConnectUrl' );
-					if ( error ) {
-						createNoticesFromResponse( error );
-						return;
-					}
-					redirectUrl = jetpackConnectUrl;
-				} )
-			);
-		}
-
 		Promise.all( promises ).then( () => {
```

Comment:
We can probably remove the `Promise.all()` usage here since there's only one promise in this case.